src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -217,6 +229,19 @@ public TokenValidationParameters()
             ValidateTokenReplay = false;
         }
 
+        /// <summary>
+        /// Gets or sets the accepted algorithms for cryptographic operations.
+        /// </summary>
+        /// <remarks>
+        /// If set to a non-empty collection, only the algorithms listed will be considered valid.
+        /// </remarks>
```

Comment:
Could this be renamed to ValidAlgorithms to better follow the naming conventions of the other properties on TokenValidationParameters?